tests/Backend/Exasol/CreateTableTest.php

```diff
@@ -2,236 +2,15 @@
 
 namespace Keboola\Test\Backend\Exasol;
 
-use Keboola\StorageApi\ClientException;
-use Keboola\StorageApi\Metadata;
-use Keboola\Test\StorageApiTestCase;
-
-class CreateTableTest extends StorageApiTestCase
+class CreateTableTest extends \Keboola\Test\Backend\CommonPart1\CreateTableTest
```

Comment:
bych to asi rebasnul nad tu moje zmeny a pustil tag rovnou s tim a tady to skipovani bych zrusil. At nemame operace navic

Reply:
Udělal sem to naopak rebasnul sem tvůj PR nad toto :) A pořešil bych další testy v dalším PR